tool/site/client/src/pages/sendPreSignPage.tsx

```diff
@@ -1,29 +1,59 @@
-import PrescriptionSummaryView, {createSummaryPrescription} from "../components/prescription-summary/prescriptionSummaryView"
+import PrescriptionSummaryView, {createSummaryPrescriptionViewProps, PrescriptionSummaryErrors} from "../components/prescription-summary/prescriptionSummaryView"
 import * as React from "react"
 import {useCallback, useContext, useEffect, useState} from "react"
 import {useCookies} from "react-cookie"
 import {Bundle, OperationOutcome} from "fhir/r4"
 import LongRunningTask from "../components/longRunningTask"
 import {AppContext} from "../index"
-import {ActionLink, Button, Label} from "nhsuk-react-components"
+import {ActionLink, Button, Form, Label} from "nhsuk-react-components"
 import ButtonList from "../components/buttonList"
 import {isBundle} from "../fhir/typeGuards"
 import {redirect} from "../browser/navigation"
 import {getResponseDataIfValid} from "../requests/getValidResponse"
 import {axiosInstance} from "../requests/axiosInstance"
 import BackButton from "../components/backButton"
+import {Formik, FormikErrors} from "formik"
+import {getMedicationRequestResources} from "../fhir/bundleResourceFinder"
+import {updateBundleIds} from "../fhir/helpers"
 
 interface SendPreSignPageProps {
   prescriptionId: string
 }
 
+interface SendPreSignPageFormValues {
+  numberOfCopies: string
+  nominatedOds: string
+}
+
+type SendPreSignPageFormErrors = PrescriptionSummaryErrors
```

Comment:
bit weird to just copy over this type, if its just to do the rename we could do that as part of the import I think

`import PrescriptionSummaryErrors as SendPreSignPageFormErrors from ...`

Reply:
interesting, I didn't spot this because the linter wrote this code for me

what i wrote was 

```interface SendPreSignPageFormErrors extends PrescriptionSummaryErrors {}```

and although it looks weird now, it's about having seperation of concerns, the Prescription Summary Component should not know the send pre sign page exists, so the Prescription summary component has its own errors, the page has it's own errors which could be a composite of component errors

Reply:
yeah the extends version make a little more sense to me but I do see why the linter has done it